webanno-ui-curation/src/main/java/de/tudarmstadt/ukp/clarin/webanno/ui/curation/component/SuggestionViewPanel.java

```diff
@@ -618,7 +618,7 @@ private void addSuggestionColor(Project aProject, Mode aMode, Map<String, JCas>
             Collection<ConfigurationSet> aCfgSet, boolean aI, boolean aAgree)
     {
         for (ConfigurationSet cs : aCfgSet) {
-            boolean use = false;
+            boolean use;
```

Comment:
I prefer having explicit initalizers for primitive variables, even if they initialize to the same value that the JVM would use.